### ISSUE TITLE
[Game] Companion developer resources for Flutter GPU

### ISSUE THREAD

**Issue by zoeyfan** [flutter/flutter#149287]

**Comment by sfshaza2** [comment]
Please let me know if this affects docs.flutter.dev for the Q3 release.

**Comment by domesticmouse** [comment]
I don't think flutter_gpu is part of Q3 release. @bdero to confirm

**Comment by bdero** [comment]
For the Q3 release/announcements, I'd prefer not to instruct users to try it out on the stable channel yet.

My goal is to resolve the MVP tasks and have it in a preview-able state by August 15th with a light call-to-action to try it out and give feedback, but on the master channel!

Codelab would be a nice-to-have, but probably not worth iterating on this early (at least not until we reach this milestone): https://github.com/orgs/flutter/projects/134/views/3
Flutter GPU is a very advanced feature and I anticipate the vast majority of users to benefit from it through higher level packages in the future.

**Comment by domesticmouse** [comment]
> Codelab would be a nice-to-have, but probably not worth iterating on this early

I usually build codelabs for features once they are heading to stable. Maybe we can build a lighter weight introduction, e.g. a medium post that describes how to build something with the feature. The advantage for this is that a medium post does not require continuing to keep the post up to date as things change.

**Comment by bdero** [comment]
@domesticmouse Sounds good. Maybe I could start drafting up something up in a shared doc to iterate on? :)

**Comment by zoeyfan** [comment]
That would be great Brandon! By the way, we have an upcoming marketing moment at Google I/O China, and we are proposing this feature to be mentioned at the keynote on Aug 7th. We would love to point to your doc if folks are looking for more guidance on 3D. @kkboateng FYI/

**Comment by bdero** [comment]
I wrote a detailed getting started blog post for the 3.24 post for this: https://medium.com/flutter/getting-started-with-flutter-gpu-f33d497b7c11

Let's track future onboarding content projects in new issues as we conceive of them!
